docs/source/release_notes.rst

```diff
@@ -2,8 +2,19 @@
 
 Release Notes
 -------------
-Future Release
-==============
+
+.. Future Release
+  ==============
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v0.24.1 May 26, 2021
+====================
     * Enhancements
```

Comment:
Unused header needs to be removed

Reply:
Same comment for Changes header

Reply:
Also, you can add yourself to the list of contributors

Reply:
Updated